easybuild/easyconfigs/g/gfbf/gfbf-2025b.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'Toolchain'
+
+name = 'gfbf'
+version = '2025b'
```

Comment:
When first adding this we use `YYYY.MM` format (so `2025.07` here) and have it as a candidate for the `2025b` toolchain. Then, when we're happy we convert it to being `2025b`. This way, if we find a big problem, we do not have some sites using `2025b` toolchains that are different to the ones that others have.